### PR TITLE
dataloaders with fast_dev_run

### PR DESCRIPTION
**Pull request by williamFalcon**
enabling no dataloaders in module with fdr

### CODE REVIEW

**Comment by awaelchli** [comment]
Nice!

File: pytorch_lightning/core/model_saving.py [hunk 1]
```diff
@@ -4,8 +4,7 @@
 """
 
 from pytorch_lightning.utilities import rank_zero_warn
+from pytorch_lightning.core.saving import *  # noqa: F403
```
Comment (Borda) [inline pytorch_lightning/core/model_saving.py:7]: Then the `# noqa: F403` is not needed

File: pytorch_lightning/core/model_saving.py [hunk 2]
```diff
@@ -4,8 +4,7 @@
...
 
 rank_zero_warn("`model_saving` module has been renamed to `saving` since v0.6.0."
```
Comment (Borda) [inline pytorch_lightning/core/model_saving.py:9]: it makes more sense to me raise the warning and then do the import...

File: tests/trainer/test_dataloaders.py [hunk 3]
```diff
@@ -11,6 +11,42 @@
...
 
 
+def test_fit_train_loader_only(tmpdir):
+
+    model = EvalModelTemplate()
+    train_dataloader = model.train_dataloader()
+
+    model.train_dataloader = None
+    model.val_dataloader = None
+    model.test_dataloader = None
```
Comment (Borda) [inline tests/trainer/test_dataloaders.py:21]: This was causing problem and was fixed in #1670

**Comment by Borda** [comment]
it seems that the formatting issues was caused by new `flake8`
https://flake8.pycqa.org/en/latest/release-notes/3.8.0.html

**Pull request merged**